File: samples/client/petstore/java/retrofit2-play26/src/main/java/org/openapitools/client/api/AnotherFakeApi.java
package org.openapitools.client.api;

import org.openapitools.client.CollectionFormats.*;
import org.openapitools.client.model.Client;
import retrofit2.Response;
import retrofit2.http.*;

public interface AnotherFakeApi {
  /**
   * To test special tags
   * To test special tags and operation ID starting with number
   * @param client client model (required)
   * @return Call&lt;Client&gt;
   */
  @Headers({
    "Content-Type:application/json"
  })
  @PATCH("another-fake/dummy")
  CompletionStage<Response<Client>> call123testSpecialTags(
    @retrofit2.http.Body Client client
  );

}

File: samples/openapi3/client/petstore/kotlin-jvm-retrofit2-coroutines/src/main/kotlin/org/openapitools/client/apis/AnotherFakeApi.kt
package org.openapitools.client.apis

import org.openapitools.client.infrastructure.CollectionFormats.*
import retrofit2.http.*
import retrofit2.Response
import okhttp3.RequestBody

import org.openapitools.client.models.Client

interface AnotherFakeApi {
    /**
     * To test special tags
     * To test special tags and operation ID starting with number
     * Responses:
     *  - 200: successful operation
     * 
     * @param client client model 
     * @return [Client]
     */
    @PATCH("another-fake/dummy")
    suspend fun call123testSpecialTags(@Body client: Client): Response<Client>

}
